docs/api/ReactWrapper/setProviders.md

```diff
@@ -0,0 +1,66 @@
+# `.setProviders(providers) => Self`
+
+Can be used to update the values provided by React's new context API. A subset of current
+providers can be passed to update just those values without removing the unspecified providers.
+
+NOTE: can only be called on a wrapper instance that is also the root instance.
+
+
+#### Arguments
+
+1. `providers` (`Array<ReactElement>`): An array of Provider elements to merge in with the current context
+
+
+
+#### Returns
+
+`ReactWrapper`: Returns itself.
+
+
+
+#### Example
+
+```jsx
+const ContextA = createContext(null);
+const ContextB = createContext(null);
+const ContextC = createContext(null);
+
+function MyComponent() {
+  return (
+    <div>
+      <ContextA.Consumer>
+        {value => <div>{value} </div>}
+      </ContextA.Consumer>
+      <ContextB.Consumer>
+        {value => <div>{value} </div>}
+      </ContextB.Consumer>
+      <ContextC.Consumer>
+        {value => <div>{value} </div>}
+      </ContextC.Consumer>
+    </div>
+  );
+}
+```
+```jsx
+const wrapper = mount(<MyComponent />, {
+  providers: [
+    <ContextA.Provider value="I" />,
+    <ContextB.Provider value="Love" />,
+    <ContextC.Provider value="Enzyme" />,
+  ],
+});
+wrapper.text(); // I Love Enzyme
+
+wrapper.setProviders([
+  <ContextA.Provider value="We" />,
+  <ContextC.Provider value="React" />,
+]);
+wrapper.text(); // We Love React
+```
+
+#### Common Gotchas
+
+- `.setProviders()` can only be used on a wrapper that was initially created with a call to `mount()`
+- `.setProviders()` is only supported in React >= 16.3
+- `.setProviders()` requires `enzyme-adapter-react-16@>=1.8` or `enzyme-adapter-react-16.3@>=1.5`
```

Comment:
These version will need to be updated depending on when/if this makes it in.

Reply:
Let's remove version numbers entirely; there's no need to specify it here.